### PR TITLE
Create basic models

### PR DESCRIPTION
**Pull request by Bartosz97**
Create basic models of database and class structure for login.

### CODE REVIEW

**Comment by jacek15157** [comment]
I'm not sure so dont be mad, but I think that UserSecretId should be in other file added to git-ignore. pls explain if it's correct (?)

**Comment by EnjoyTheNoise** [comment]
Some general remarks:
- do we need all those empty classes / folders now?
- overall suggestion: manage usings in files c:

File: src/Joblify.Core.Api/Startup.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc();
+            services.AddMvc();
```
Comment (EnjoyTheNoise) [inline src/Joblify.Core.Api/Startup.cs]: I believe we can manage w/o this tab here 🤔

File: src/Joblify.Core.Data/Models/User.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+        [Required]
+        [ForeignKey("Role")]
+        public byte RoleId { get; set; }
+
+        public Role Role { get; set; }
+
+        public bool IsDeleted { get; set; }
+
+
```
Comment (EnjoyTheNoise) [inline src/Joblify.Core.Data/Models/User.cs]: formatting here

**Review by kobrynsky — APPROVED** [review]
imho everything is ok right now C:

**Comment by JJMPdsk** [comment]
Looks fine

**Review by JJMPdsk — APPROVED** [review]

**Review by jacek15157 — APPROVED** [review]

**Pull request merged**